README.md

```diff
@@ -1,2 +1,53 @@
-# rhcos-slb
-Config scripts for RHCOS SLB 
+# Setting up network bonding on specific interfaces during the ignition process
+
+## Overview 
+This guide explains how to bond specific NICs during the initial OpenShift installation with OVS and supplies the Ignition and scripts to achieve this.
+
+## Important
+This guide describes the customisations needed for installation when using a PXE server.
+
+## Prerequisites
+- [Fcct](https://github.com/coreos/butane) software to be able to convert fcc to ign files.
+- PXE server with ability to set kernel arguments.
+- Web accessible location (HTTP only) to upload the custome ign files.
+
+## Usage  
+
+### 1. Adding parameters to PXE server
+Key   |  Type | Value | Exmaple |Discription
+---   | --- | --- | --- | --- |
+ignition.config.url  | String | http://{location reachable to PXE and remote VM in order to get the ignition file}/```file.ign``` custom-config | ignition.config.url=```http://my.web.server.redhat.com/file.ign``` custom-config | This parameter is used to allow the server to load an extra ignition file which captures the MAC addresses and saves them into a file for later use in  machine config.
+macAddressList | String | List of MAC address should be set per host In the following order:  1. Primary NIC  2. Secondary NIC  3. All the rest (optional)| 00:01:02:03:04:05,06:07:08:09:10:11 ... | This will be the list of MAC addresses which exist on the host and will be grabbed by the ignition file.
+
+
+### 2. Prepare the files
+- Upload `file.ign` to a shared location which the OpenShift nodes can access.
```

Comment:
perhaps we should add another bullet here that creates the `file.ign` file in the first place:
For example:
- Create the ign file from the custom-config.fcc:
```
fcct custom-config.fcc > file.ign
```

Reply:
Done